.dockerignore

```diff
@@ -0,0 +1,2 @@
+debug/
+target/
```

Comment:
Please add a newline at the end of this file.

Reply:
done